MaterialDesign3.Demo.Wpf/Domain/SampleItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MaterialDesignThemes.Wpf;
+
+namespace MaterialDesign3Demo.Domain;
+
+public class SampleItem
+{
+    public string Title { get; set; }
```

Comment:
Because nullable reference types is enabled, this should be declared as a nullable string (this is the build failure on the pipeline; only a warning when compiled locally)
```suggestion
    public string? Title { get; set; }
```